.gitignore

```diff
@@ -38,3 +38,4 @@ standalone-metastore/metastore-server/src/gen/version
 launch.json
 settings.json
 kafka-handler/src/test/gen
+
```

Comment:
unused can be removed

Reply:
Is the empty udf_split_map_privs.q.out file needed?

Reply:
> Is the empty udf_split_map_privs.q.out file needed?
no, it is generated while building